changelog/12633.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+auth/kubernetes: Add ability to configure entity alias names based on the serviceaccount's namespace and name
+```
```

Comment:
Should this include links to the relevant PRs in the k8s auth repo? Similar to https://github.com/hashicorp/vault/pull/12600/files.

Reply:
Updated in https://github.com/hashicorp/vault/pull/12633/commits/360d515fedd890d6893cb01e026d153f95c2079b